### PR TITLE
[Junos]: Fixes needed for Sanity

### PR DESCRIPTION
**Pull request by rohitthakur2590**
##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: tests/sanity/ignore-2.9.txt [hunk 1]
```diff
@@ -1,3 +1,6 @@
+plugins/action/junos.py action-plugin-docs # base class for deprecated network platform modules using `connection: local`
+plugins/doc_fragments/junos.py future-import-boilerplate
+plugins/doc_fragments/junos.py metaclass-boilerplate
 plugins/module_utils/network/junos/argspec/facts/facts.py future-import-boilerplate
 plugins/module_utils/network/junos/argspec/facts/facts.py metaclass-boilerplate
```
Comment (pabelanger) [inline tests/sanity/ignore-2.9.txt:5]: looks like some lines in this file can be removed

https://0c5ee2747a958bc92a97-77212082f2de2f22f87ec6cf98595a55.ssl.cf5.rackcdn.com/60/e11d17aaa09b8bfd8f97c1641b2929eec667a821/check/ansible-test-sanity-junos/3f3daa2/job-output.html#l10067

**Comment by rohitthakur2590** [comment]
recheck

**Comment by rohitthakur2590** [comment]
recheck

**Comment by rohitthakur2590** [comment]
recheck

**Comment by rohitthakur2590** [comment]
recheck

**Pull request merged**